### ISSUE TITLE
not connect in match

### ISSUE THREAD

**Issue by Kapuchinko** [ValveSoftware/Dota2-Gameplay#15463]
### Ability name

all types of game selections

### Description

Hello dear developers, after your new patch. Many players (most of these people on Windows 11) have such a problem as not connecting to the found match. Someone just has a mistake, and someone has an endless mistake. Simply put, there is no stable connection. (from your side. Tested on different devices). only the genuinely lucky managed to play at least something

### Example Match ID (and possibly Timestamp)

.

**Comment by montefuscojm** [comment]
YES VALVE PLEASE HELP! im on a 45 minute ban just finished a game and queue for the next and instant ban.
